browser/brave_shields/ad_block_subscription_download_manager_getter.cc

```diff
@@ -80,12 +80,20 @@ class AdBlockSubscriptionDownloadManagerGetterImpl
         profile_manager->user_data_dir().Append(
             profile_manager->GetInitialProfileDir()));
     if (profile) {
-      GetDownloadManager(profile);
+      base::ThreadTaskRunnerHandle::Get()->PostTask(
+          FROM_HERE,
+          base::BindOnce(
+              &AdBlockSubscriptionDownloadManagerGetterImpl::GetDownloadManager,
+              base::Unretained(this), profile));
```

Comment:
`base::Unretained(this)` is safe, but we can't guarantee the lifetime of `profile` here

Reply:
fixed